README.md

```diff
@@ -1,3 +1,5 @@
-# pb026393.github.io
-
-SMART ON FHIR STARTER APP
+# pb026393.github.io
+
+SMART ON FHIR STARTER APP
+

```

Comment:
Let's use this README to add project specific information such as what this project is for, how to get started, and how to run test.

Then you can link to the tutorial below for more detail.